stable/selenium/values.yaml

```diff
@@ -53,6 +53,11 @@ hub:
   ## Path for checking readiness and liveness via HTTP Request
   probePath: "/wd/hub/status"
 
+  # Configure security context on the hub pod
```

Comment:
Do we need to leave these values as a default?
Can we leave them commented out and allow each person to enable them on their own?

Reply:
I am not actually sure about the best "non breaking change" default to put here.
The underlying image is using 1000 as a user, my understanding is that putting here anything else would be a breaking change. 
But I may be wrong. Let me know what you think.